packages/core/src/Stores/client-store.js

```diff
@@ -615,6 +618,129 @@ export default class ClientStore extends BaseStore {
+            },
+            USD: {
+                dark: 'IcWalletCurrencyUsd',
+                light: 'IcWalletCurrencyUsd',
+            },
+            EUR: {
+                dark: 'IcWalletCurrencyEur',
+                light: 'IcWalletCurrencyEur',
+            },
+            AUD: {
+                dark: 'IcWalletCurrencyAud',
+                light: 'IcWalletCurrencyAud',
+            },
+            GBP: {
+                dark: 'IcWalletCurrencyGbp',
+                light: 'IcWalletCurrencyGbp',
+            },
+            BTC: {
+                dark: 'IcWalletBitcoinDark',
+                light: 'IcWalletBitcoinLight',
+            },
+            ETH: {
+                dark: 'IcWalletEthereumDark',
+                light: 'IcWalletEthereumLight',
+            },
+            USDT: {
+                dark: 'IcWalletTetherDark',
+                light: 'IcWalletTetherLight',
+            },
+            eUSDT: {
+                dark: 'IcWalletTetherDark',
+                light: 'IcWalletTetherLight',
+            },
+            tUSDT: {
+                dark: 'IcWalletTetherDark',
+                light: 'IcWalletTetherLight',
+            },
+            UST: {
+                dark: 'IcWalletTetherDark',
+                light: 'IcWalletTetherLight',
+            },
+            LTC: {
+                dark: 'IcWalletLiteCoinDark',
+                light: 'IcWalletLiteCoinLight',
+            },
+            USDC: {
+                dark: 'IcWalletUsdCoinDark',
+                light: 'IcWalletUsdCoinLight',
+            },
+        };
+
+        return this.all_loginids
+            .filter(id => this.getAccount(id)?.account_category === 'wallet')
+            .map(id => {
+                const account = this.getAccount(id);
+                const loginid = id;
+                const currency = account.currency;
+                const is_disabled = Boolean(account.is_disabled);
+                const is_virtual = Boolean(account.is_virtual);
+
+                const fiat_or_crypto = this.is_crypto(currency) ? 'crypto' : 'fiat';
+                const icon_type = is_virtual ? 'demo' : fiat_or_crypto;
+                const landing_company_name = account.landing_company_name?.replace('maltainvest', 'malta');
+                const is_malta_wallet = landing_company_name === 'malta';
+                const dtrade_loginid = account?.linked_to?.find(account => account?.platform === 'dtrade')?.loginid;
+                const dtrade_balance = this.accounts?.[dtrade_loginid]?.balance;
+
+                const wallet_currency_type = is_virtual ? 'Demo' : currency || '';
+                const icons = currency_to_icon_mapper[wallet_currency_type];
+
+                const gradients = {
+                    /** The gradient class name for the wallet header background. */
+                    header: {
+                        dark: `wallet-header__${wallet_currency_type.toLowerCase()}-bg--dark`,
```

Comment:
minor: generally store shouldn't be aware of css classes / BEM / modifiers etc, that should stay within component responsibility (though I guess there is a reason for putting it here)

Reply:
in my mind, ideally, component should receive currency, theme and all basic parameters to generate its own clasess and modifiers

Reply:
@wojciech-deriv I understand what you mean, but I'm not sure I agree with in this case. I just copied this logic from `useWalletAccountsList` because we have the same classes for all wallet cards and wallet headers. We added these classes description to `app/packages/core/src/sass/app/_common/components/wallet.scss` and uses everywhere. Yes, now when we have wallets as separated package, maybe we don't need this, but for dropdown I need to use core package and in my opinion it's not a bad solution to define classname inside of the computed property and just use it outside instead of create the class name in every component based on `wallet_currency_type`, `currency` and `theme`. Just IMHO. But if you are pretty sure it's a bad approach, I'll change it, not problem at all. Let me know

Reply:
I'm sure its bad pattern to use classes directly within store. Store is responsible for logic, not visual aspects. If component gets refactored to new one, store shoduln't require changes. 

I commented mainly for future considerations - I do not see a point in refactoring it now, especially when stores are effectively deprecated :D but would be good to avoid carrying over legacy patterns or anything build in future

Reply:
@wojciech-deriv In this case I'm pretty sure we just need to add one more layer (earlier it was a hook) where we can create common things and use it inside of the any components. Agree with you that the store is not the right place for this, but still sure if we have the same classes for many components (walletModal, walletCard, walletIcon, walletAppIcon, successModal and next) would be better to create these common classes in one place instead of creating it in every component yourself

Reply:
yes, agree, would be good to encapsulate specific concerns,

to encapsulate data/logical concern, typically the way to go is to use hooks (at least within wallets v2)
to encapsulate visual concerns, typically the way to go is to create compnent (e.g. <CurrencyGradient currency="GPB">{children}</Currency>)

if you want to have universal clasess - then it is a valid option to consider, but they shouldn't be component specific like existing ones: wallet-card__gbp-bg--dark or wallet-header__gbp-bg, they should be concern specific, e.g. "gradient-gpb-dark"

Reply:
@wojciech-deriv about your specific example `gradient-gpb-dark` , the main problem is for GBP wallet header it will be gradient-gbp, but for GBP wallet card it will be another gradient-gbp, so you need at least 2 types of gradients for every currency for light theme and 2 classes for dark theme for every currency. This is the main reason why we call it `wallet-header-gradient-gbp-dark` or `wallet-card-gradient-gbp-dark`. In my opinion, put all needed information inside of the hook and return all needed data for UI as a result is a better than create class name in the component.

IMHO it's concern specific `wallet-header-...` or `wallet-card-...`, or we can just name it `gradient-gbp-one` and `gradient-gbp-two`, because we have at least 2 types of gradients for wallets.

Or maybe I understood you wrong, but in my opinion put data (logic, icon name and even class if it's the list of something and every item has own UI ) inside of the hook is a good. Again, agree with you about store, it's not the place for this, but hooks fit this purpose. Let me know your final decision and I'll refactor the code if needed.

Reply:
I'm happy to proceed and merge as it is - no point in refactoring something now when store is deprecated anyway, especially given those are existing patterns within stores. 

In future/new functionalities - this is where we really should establish good patterns of handling data and visual aspects (especially, establish whats responsible for handling data and whats responsible for visual aspects - to have some degree of separation between those two concerns).

Reply:
We should not be adding more load to client-store. Lets just go with our hooks in hooks package.

Reply:
@thisyahlen-deriv Ok, got your point. In this case I'll refactor it soon and update the PR.